translator/totomlconfig/sampleConfig/prometheus_config_windows.json

```diff
@@ -63,8 +63,10 @@
             },
             {
               "label_matcher": "default",
+              "metric_namespace": "CustomizeMetricScrappingJobNamespace",
               "metric_selectors": [".*"],
               "source_labels": ["Namespace"]
+
```

Comment:
this newline is not necessary

Reply:
Nice catch. Fixed it in the newest commit.